### PR TITLE
[python-api] add some get-methods to xbmc.InfoTagVideo

### PR DESCRIPTION
**Pull request by phil65**
Allows to fetch the media type from a listitem.

@tamland 

EDIT: added some more getter-methods. They become more important once contextmenu can get opened at home (up to now using getInfoLabel("Listitem.xxx") worked fine for context item add-ons since it was used exclusively in MediaWindows, but for home we need to use sys.listitem)

### CODE REVIEW

File: xbmc/interfaces/legacy/InfoTagVideo.h [hunk 1]
```diff
@@ -74,6 +74,10 @@ namespace XBMCAddon
       /**
        * getVotes() - returns a string.
        */
+      String getMediaType();
+      /**
+       * getMediaType() - returns a string.
```
Comment (xhaggi) [inline xbmc/interfaces/legacy/InfoTagVideo.h]: you did something wrong with c/p of the comment block

Comment (phil65) [inline xbmc/interfaces/legacy/InfoTagVideo.h]: yup already noticed... fixed.

**Comment by tamland** [comment]
+1. Remember to bump api version though

**Comment by phil65** [comment]
I think we only bump once every release for python?  @ronie

**Comment by tamland** [comment]
I think we never truly found a good solution for that.. The problem is that without it, there is no way scripts to start using this feature or write compatibility code (without resorting to hasattr)

**Comment by ronie** [comment]
we have no strict rules for it.
sometimes we bump just once within a release cycle,
other times we bump a couple of times.

**Comment by phil65** [comment]
> The problem is that without it, there is no way scripts to start using this feature

Well, any script which gets pushed to our krypton branch from addon repo may use these features?

Lookin at recent past, I dont think this needs a dedicated API bump.
If anyone disagrees I can still do it of course. :)

**Comment by tamland** [comment]
Go for whatever. I don't know if anybody even cares :)

**Comment by phil65** [comment]
jenkins build this please.

**Comment by phil65** [comment]
adjusted method name as suggested by @xhaggi.
jenkins build this please.

**Pull request merged**